azure_jumpstart_arcbox/ARM/azuredeploy.json

```diff
@@ -88,14 +92,18 @@
     "capiTemplateUrl": "[uri(variables('templateBaseUrl'), 'ARM/kubernetes/ubuntuCapi.json')]",
     "clientVmTemplateUrl": "[uri(variables('templateBaseUrl'), 'ARM/clientVm/clientVm.json')]",
     "mgmtTemplateUrl": "[uri(variables('templateBaseUrl'), 'ARM/mgmt/mgmtArtifacts.json')]",
-    "mgmtStagingStorageUrl": "[uri(variables('templateBaseUrl'), 'ARM/mgmt/mgmtStagingStorage.json')]"
+    "mgmtStagingStorageUrl": "[uri(variables('templateBaseUrl'), 'ARM/mgmt/mgmtStagingStorage.json')]",
+    "keyVaultTemplateUrl": "[uri(variables('templateBaseUrl'), 'ARM/key-vault/key-vault.json')]"
```

Comment:
We are using camel case for other folder names so I suggest we keep that convention here and change the path from key-vault to keyVault.

Reply:
BTW, "keyVault" is Lower Camel Case, whereas "KeyVault" is Camel Case. Just a note on nomenclature.

Reply:
done